src/Math/Measures.dfy

```diff
@@ -134,6 +134,13 @@ module Measures {
     requires set1 <= set2
     ensures mu(set1) <= mu(set2)
 
+  lemma {:axiom} MeasureOfDisjointUnionIsSum<T>(eventSpace: iset<iset<T>>, mu: iset<T> -> real, set1: iset<T>, set2: iset<T>)
```

Comment:
I thought it would be good to have a version that takes arguments since the guidelines say to avoid `forall`s in the postcondition. But I added a proof of this lemma.

Reply:
Thanks, and yes, if you want to keep this lemma please add a proof that makes use of `IsAdditive`.